scripts/download_cams_pm25.py

```
@@ -0,0 +1,267 @@
+        
+        logger.info(f"Downloading data for {len(locations)} locations")
+        logger.info(f"Parameters: {parameters}")
+        logger.info(f"Period: {start_date.date()} to {end_date.date()}")
+        logger.info(f"Data type: {'Forecast' if use_forecast else 'Reanalysis'}")
+        
+        # Create output directory
+        output_dir = output_dir or Path(f"data/cams/processed")
+        output_dir.mkdir(parents=True, exist_ok=True)
+        
+        # Prepare CSV file
+        timestamp_str = datetime.now().strftime('%Y%m%d_%H%M%S')
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)